vignettes/Data_retrieval_via_API.Rmd

```diff
@@ -0,0 +1,508 @@
+
+```{r plot_aeid_spid, eval = FALSE}
+# load all matching spids and then subset using the aeid desired to find m4id
+mc5 <- tcplLoadData(lvl = 5, 
+                    fld = "spid",
+                    val = "TP0000904H05",
+                    type = "mc", 
+                    add.fld = FALSE) # 8 rows of data
+m4id <- mc5[aeid == 714]$m4id # subset to 1 aeid extract m4id
+
+# default parameters used here: fld = "m4id", type = "mc" (type can never be "sc" when connected to API)
+tcplPlot(val = m4id, output = "jpg", verbose = TRUE, flags = TRUE)
+```
+
+<center>![<font style="font-size:15px"><i>API-sourced plot with parameters: output = "jpg", verbose = TRUE, and flags = TRUE for m4id 1847540</i></font>](img/API_plot_1847540.jpg)</center>
+
+Use the "compare" feature to supply more m4ids to `compare.val` and compare them one to one with the m4ids in `val`. `val` and `compare.val` must be the same length!
+
+```{r plot_m4id_compare, eval = FALSE}
+# using the data pulled in the previous code chunk 'mc5'
+m4id <- mc5$m4id # create m4id vector length == 8
+
+# default parameters used here: fld = "m4id", type = "mc" (type can never be "sc" when connected to API)
+tcplPlot(val = m4id[1:4], 
+         compare.val = m4id[5:8], 
+         output = "pdf", 
+         verbose = TRUE, 
+         multi = TRUE, 
+         flags = TRUE, 
+         yuniform = TRUE,
+         fileprefix = "API_plot_compare")
+```
+
+<center>![<font style="font-size:15px"><i>API-sourced plots with parameters: output = "pdf", verbose = TRUE, multi = TRUE, flags = TRUE, and yuniform = TRUE for m4ids 1833668, 1834094, 1836233, and 1836494, compared with m4ids 1839401, 1842443, 1847540, and 1850045</i></font>](img/API_plot_compare.png)</center>
+
+Supply `fld = "aeid"` to plot every curve available in the API for the given endpoint(s).
+
+```{r plot_aeid, eval = FALSE}
+# plot all curves across endpoint(s)
+tcplPlot(fld = "aeid", 
+         val = 704, 
+         output = "pdf", 
+         verbose = TRUE, 
+         multi = TRUE, 
+         yrange = c(-100,100),
+         fileprefix = "API_plot_704")
+```
+
+<center>![<font style="font-size:15px"><i>API-sourced plots with parameters: output = "pdf", verbose = TRUE, multi = TRUE, and yrange = c(-100,100) for aeid 704</i></font>](img/API_plot_704.png)</center>
+
+## Standalone Format Plot with API
+
+Suppose you have a database where you process dose-response data and wish to make comparison to how it was previously processed in invitrodb v4.1. The API connection supports <font face="CMTT10">tcplPlot</font>'s standalone plotting feature! Using the utility function <font face="CMTT10">tcplPlotLoadData</font> while connected to the API, you can pass along that data to <font face="CMTT10">tcplPlot</font>'s `dat` parameter and use `compare.val` for the database data.
```

Comment:
I think "make comparison" on line 483 should be either "make a comparison" or "make comparison".